lib/repl.js

```diff
@@ -778,6 +779,19 @@ function filteredOwnPropertyNames(obj) {
   return Object.getOwnPropertyNames(obj).filter(intFilter);
 }
 
+function getGlobalLexicalScopeNames() {
+  inspector.open();
+  const session = new inspector.Session();
+  session.connect();
```

Comment:
This will conflict with any existing open connections, e.g. when the user is experimenting with the `inspector.Session` themself in the REPL. Last I heard @eugeneo was working on multi-connection Inspector support but even it is now supported in V8, the Node.js part isn't ready yet.

```js
> new (require('inspector').Session)().connect()
undefined
> new (require('inspector').Session)().connect()
TypeError: Session is already attached
    at Session.connect (inspector.js:27:9)
    at repl:1:38
```